src/components/pdf-highlighter/utils/rects.js

```diff
@@ -0,0 +1,81 @@
+import { sortBy } from 'lodash'
+
+// TODO: allow to highlight when PDF is rotated
+const normalizeToScale1 = ({ top, left, width, height }, scale) => {
```

Comment:
Why cannot you normalize scale to be in `[0..1]` and then multiply once without repeating yourself?

Reply:
Do you mean this?

```
const normalizeToScale1 = ({ top, left, width, height }, scale) => {
  const normalizedScale = 1 / scale

  return {
    top: top * normalizedScale,
    left: left * normalizedScale,
    width: width * normalizedScale,
    height: height * normalizedScale,
  }
}
```

That's actually a good idea. I created a commit for it which will be part of another PR. Thanks